lightning/livekit_example/generate_audio.py

```diff
@@ -0,0 +1,37 @@
+import asyncio
+import aiohttp
```

Comment:
import aiohttp
ModuleNotFoundError: No module named 'aiohttp'

need to add to requirements.txt